### PR TITLE
Add contributor information to README

### PR DESCRIPTION
**Pull request by aidigii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+
+## Getting Started
+
+Stable release: checkout [master](https://github.com/acmutd/comet_planning/tree/master)
+
+Developer version: checkout [dev](https://github.com/acmutd/comet_planning/tree/dev)
+
+## Git Work Flow
+
+There will be three branches that we will be utilizing: the master branch, the dev branch, and branches for particular features. Below are the step by step process we will use as a team.
```
Comment (WillieCubed) [inline README.md:13]:
```suggestion
This project loosely uses the [Git Flow](https://nvie.com/posts/a-successful-git-branching-model/) branching model.

There are three types of branches: `master`, which always contains complete, production-ready copies of project code, `dev`, which is guaranteed to at least be runnable on any machine (generally free of bugs), and feature branches, which are children of `dev` that are primarily used to create new features.

Here is the process to make changes to the project:
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,35 @@
-# comet_planning
-A tool for the UTD community to plan out their coursework. 
+# Comet_Planning
+
+ A drag-and-drop interface to help UTD students plan out their 4-year coursework.
```
Comment (WillieCubed) [inline README.md:3]:
```suggestion
 **A drag-and-drop interface to help UTD students plan out their 4-year coursework.**
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,35 @@
-# comet_planning
-A tool for the UTD community to plan out their coursework. 
+# Comet_Planning
```
Comment (WillieCubed) [inline README.md:1]:
```suggestion
# Comet Planning
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,35 @@
...
+
+ A drag-and-drop interface to help UTD students plan out their 4-year coursework.
+
+## Getting Started
+
```
Comment (WillieCubed) [inline README.md:6]:
```suggestion
This project requires a working Node.js and NPM installation.

To start, clone the repository, and then run `npm start` to launch a local development server.
```

**Comment by aidigii** [comment]
Suggested changes made on README file.

**Pull request closed without merging**